core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -336,12 +336,41 @@ var ExplorationEditorMainTab = function() {
         var ruleDescription = _getRuleDescription(interactionId, ruleName);
         // Replace selectors with feedbackTextArray's elements.
         ruleDescription = _replaceRuleInputPlaceholders(
-          ruleDescription, feedbackTextArray);
+          ruleDescription, feedbackTextArray, interactionId);
         ruleDescription += '...';
         // Adding "..." to end of string.
         var answerTab = element(by.css('.protractor-test-answer-tab'));
         expect(answerTab.getText()).toEqual(ruleDescription);
       },
+      /**
+       * Check for correct rule parameters in the case of multiple rules in an
+         answer group.
+       * @param {string[][]} - A 2 dimensional array , where each 1 dimensional
+         array contains the parameters for a particular rule.
+       */
+      expectMultipleRulesToBe: function(ruleParamatersArray) {
```

Comment:
"parameters" is misspelled, here and elsewhere.

Reply:
done